pkgs/by-name/me/meson/package.nix

```diff
@@ -87,11 +87,10 @@ python3.pkgs.buildPythonApplication rec {
 
     # Fix meson's detection for zig's linker
     # https://github.com/mesonbuild/meson/pull/12293
-    (fetchpatch {
-      name = "linker-support-zig-cc.patch";
-      url = "https://github.com/mesonbuild/meson/pull/12293/commits/2baae244c995794d9addfe6ed924dfa72f01be82.patch";
-      hash = "sha256-dDOmSRBKl/gs7I3kmLXIyQk3zsOdlaYov72pPSel4+I=";
-    })
+    ./linker-support-zig-cc.patch
+    # Fix meson's tests for zig's linker
+    # https://github.com/mesonbuild/meson/pull/13493
+    ./linker-support-zig-cc-tests.patch
```

Comment:
Would fetchpatched backports apply?

Reply:
Yeah.  (But it's also fine to just drop this change and wait for the next Meson upgrade at this point, if you prefer.)